README.md

```diff
@@ -61,12 +61,6 @@ To provide your own user list, replace the number on line 9 of the `gatsby-confi
 
 Make sure that the account provided is following at least one account and that Twitter ID on line 9 is in quotes.
 
-### Nomination
-
-In order to facilitate the nominations of new engineers to follow, the nominate page allows users to send a DM directly to the twitter account of your choice.
-
-On line 16 of the `nominate.js` file, replace the variable `twitterIdForFollowingList` with the twitter id of your desired account.
-
```

Comment:
we should probably add this back in at some point; I didn't notice previously that reverting the commit removed this feature